aks/redis/variables.tf

```
@@ -95,6 +95,11 @@ variable "azure_enable_monitoring" {
   default = true
 }
 
+variable "azure_redis_replica_zones" {
```

```suggestion
variable "azure_zones" {
```

We don't need `redis` as we're already within the Redis module.

I'd also suggest `azure_zones` without `replica` so it matches the name of the argument we pass in to the`azurerm_redis_cache` resource, just makes it clear that we're not doing anything special with the argument, but I don't really mind on that.